### PR TITLE
Adds basic text for components, MD library for documentation

### PR DESCRIPTION
**Pull request by rvsia**
Just basic support for adding custom text to components examples

Also, I tried to add a library for using MD on the page:

I found https://github.com/rexxars/react-markdown which is very easy to use, however I don't know how easy will be to 'restyle' it to something what will more resemble MUI. Or we could try to use https://github.com/showdownjs/showdown , which looks as more detailed option.

### CODE REVIEW

**Comment by Hyperkid123** [comment]
@rvsia nice job. Its a great start. We will have to add a ton of editable examples especially for the component mapping but this looks really good.

**Comment by rvsia** [comment]
@Hyperkid123 great!

And I refactored notes to component, which allow us to change them according to mapper. (Right now, it just changes the link for additional docs. :book: )

File: src/editor-demo/common/doc-texts/installation.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactMarkdown from '../md-helper';
+
+const text =  `\`\`\`
```
Comment (Hyperkid123) [inline src/editor-demo/common/doc-texts/installation.js]: this should be language sh

File: src/editor-demo/common/md-helper/index.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+
+const renderers = {
+  paragraph: ({ children }) => <Typography variant="body1">{ children }</Typography>,
+  code: ({ value }) =>  <div style={{ background: '#272822', paddingTop: 5, paddingBottom: 5, marginTop: 10, marginBottom: 10 }}>
+    <AceEditor
+      readOnly
+      mode="jsx"
```
Comment (Hyperkid123) [inline src/editor-demo/common/md-helper/index.js]: we shoudl change mode base on language. You can get language as a prop from the component
```js
code: ({ code, language }) => ...
```

**Pull request merged**